common/clap_utils/Cargo.toml

```diff
@@ -7,9 +7,12 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-clap = "2.33.3"
+clap = "3.0.4"
 hex = "0.4.2"
 dirs = "3.0.1"
+ethereum-types = "0.12.1"
 eth2_network_config = { path = "../eth2_network_config" }
 eth2_ssz = "0.4.1"
-ethereum-types = "0.12.1"
+serde_yaml = "0.8.21"
```

Comment:
if there is no downside, could we use `"0.8.13"`, It's what we have everywhere so we could save shipping an additional crate

Reply:
Good call I went with updating the dependency everywhere else though

Reply:
even better :)